src/Game/Consol3Game.cpp

```diff
@@ -64,7 +64,17 @@ namespace Game
 			.SetRotation(Angle(-90, 0, 0));
 
 		mesh = StaticMesh();
-		mesh.SetModelResource("res/face.obj").SetTextureResource("res/normalmap.bmp");
+		mesh.SetModelResource("res/face.obj")
+			.SetTextureResource("res/normalmap.bmp")
+			.SetNormalMapResource("res/normalmap.bmp")
+			.SetPosition(Vector3(-1, 0, 0))
+			.SetRotation(Angle(0, 3.14159f / 2 * 2, 0));
```

Comment:
Ahhem, I think you'll find it's `3.1415926535897932384629` :V

Reply:
```cpp 
static const int pi = 3;
```

I think that should be good right? 😄

Reply:
Ship it!